quickwit/quickwit-ingest/src/ingest_v2/ingester.rs

```diff
@@ -1054,12 +1054,12 @@ impl Ingester {
             Err(_) => {
                 return json!({
                     "status": "initializing",
-                    "shards": [],
+                    "shards": {},
```

Comment:
(not for this PR): this kind  of change can easily be wrong (as in not all codepath were updated), in the future i think we want a debug_info_internal which returns a struct, and use `serde_json::to_value` to generate the type-erased value inside debug_info()